cns/NetworkContainerContract.go

```diff
@@ -220,3 +230,36 @@ type UnpublishNetworkContainerResponse struct {
 	UnpublishStatusCode   int
 	UnpublishResponseBody []byte
 }
+
+// ValidAclPolicySetting - Used to validate ACL policy
+type ValidAclPolicySetting struct {
+	Protocols       string `json:","`
+	Action          string `json:","`
+	Direction       string `json:","`
+	LocalAddresses  string `json:","`
+	RemoteAddresses string `json:","`
+	LocalPorts      string `json:","`
+	RemotePorts     string `json:","`
+	RuleType        string `json:","`
+	Priority        uint16 `json:","`
+}
+
+// Validate - Validates network container request policies
+func (networkContainerRequestPolicy *NetworkContainerRequestPolicies) Validate() error {
+	// validate ACL policy
```

Comment:
As one of my comments said can we enforce the validation on apipa endpoint? Check if the provided acl is on apipa endpoint type, if not fail the validation because we don;t have support for acls on cust subnet endpoint. When we add that we should remove the enforced apipa endpoint validation you'll add here. Let me know if it makes sense / you want to discuss this further.

Reply:
Got it, restricted this API only to APIPA in latest commit.